crates/bitwarden/Cargo.toml

```
@@ -52,6 +52,7 @@ getrandom = { version = ">=0.2.9", features = ["js"] }
 
 bitwarden-api-identity = { path = "../bitwarden-api-identity", version = "0.2.0" }
 bitwarden-api-api = { path = "../bitwarden-api-api", version = "0.2.0" }
+chrono = { version = "0.4.26", features = ["serde"] }
```

Move up to the other dependencies. Use loose version and turn off default-features since we don't want `oldtime`.

```suggestion
chrono = { version = ">=0.4.26, <0.5", features = ["serde"], default-features = false }
```